src/python/grpcio/grpc/_channel.py

```diff
@@ -785,10 +801,13 @@ def _poll_connectivity(state, channel, initial_try_to_connect):
 
 
 def _subscribe(state, callback, try_to_connect):
+  def cancel_all_subscriptions(timeout):
```

Comment:
Nitpicks, but:

(1) replace the body of this function with `_moot(state)`. Controlling principle: there will be one less site in the code where `state` is mutated.
(2) move this definition down and to the right so that code paths through `_subscribe` that do not use this function also do not define it. Controlling principle: "Effective Java, Second Edition", Item 45: "Minimize the scope of local variables".